portfolio/src/main/webapp/index.html

```diff
@@ -68,19 +73,51 @@ <h2>Nicole's Gallery</h2>
             <li>One Direction!!!!!</li>
         </ul>
      
+        <!--Button to generate random thoughts-->
         Button that generates sayings I normally say.
       <p>Click here to see what one of my many catchphrases:</p>
+      <!--Creates button from container-->
       <button onclick="addRandomGreeting()">My Thoughts</button>
       <div id="greeting-container"></div>
     </div>
   </body>
 
+  <!--Generat random movies-->
   <body>
-      <h2> Message Example</h2>
+      <h2> Some of My Favorite Movies</h2>
       <p>Click this button to see the server's message.</p>
+      <!--Button for random movies-->
       <button onclick="randomMovie();">Movies</button>
       <div id="movie-container"></div>
+      <!--link to entire server response-->
       <p>Click <a href="/hello"> here</a> to see the server response directly.</p>
     </body>
+
+<!--Translate about me section to language of choice-->
+<body>
```

Comment:
I overlooked this before, but you can use a single pair of `<body>` tags to wrap the entire content of this page instead of having multiple pairs of `<body>` tags. Then, to wrap sections of content, you would use the `<div>` tag.